packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/Choices.js

```diff
@@ -35,18 +37,18 @@ export function Choices({
   task,
   theme = defaultTheme
 }) {
-  const [focusIndex, setFocusIndex] = useState(0)
+  const [focusIndex, setFocusIndex] = useState(null)
 
   useEffect(() => {
     if (selectedChoiceIds.length > 0) {
       const lastSelectedIndex = filteredChoiceIds.indexOf(selectedChoiceIds[selectedChoiceIds.length - 1])
       if (lastSelectedIndex > -1) {
         setFocusIndex(lastSelectedIndex)
       } else {
-        setFocusIndex(0)
+        setFocusIndex(null)
       }
     } else {
-      setFocusIndex(0)
+      setFocusIndex(null)
```

Comment:
When the focus index is null, where does keyboard focus go?

Reply:
It appears to go back to the `body` when I test on app-project https://local.zooniverse.org:3000/projects/markb-panoptes/survey-task-test-project/classify/workflow/2434 or lib-classifier https://local.zooniverse.org:8080/?project=1634&workflow=2434 , then when I tab on app-project keyboard focus goes to the Filter button.

Reply:
Is that a problem?